src/MovieDetails.js

```diff
@@ -2,9 +2,7 @@ import React from 'react';
 import './css/MovieDetails.css'
 
 const MovieDetails = (props) => {
```

Comment:
I love that you made this a functional component since it didn't need it's on state or anything. While this might change depending on how you implement router, it's a great choice for now!